docs/index.html

```diff
@@ -0,0 +1,229 @@
+          >
+            <div class="flex-grow text-center mt-6 sm:mt-0">
+              <h1
+                class="title-font font-bold sm:text-4xl text-3xl mb-16 font-medium text-green-500"
+              >
+                Features
+              </h1>
+              <h2 class="text-green-500 text-lg title-font font-bold mb-2">
+                Moderation
+              </h2>
+              <p class="leading-relaxed text-base">
+                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Quis
+                culpa nesciunt iusto facere quam laboriosam deserunt omnis sunt
+                reiciendis distinctio quia, dolorum molestiae placeat velit,
+                recusandae sint, necessitatibus odio accusamus!
+              </p>
+            </div>
+          </div>
+          <div
+            class="flex items-center lg:w-3/5 mx-auto border-b pb-10 mb-10 border-gray-800 sm:flex-row flex-col"
+          >
+            <div class="flex-grow text-center mt-6 sm:mt-0">
+              <h2 class="text-green-500 text-lg title-font font-bold mb-2">
+                Music
+              </h2>
+              <p class="leading-relaxed text-base">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Modi
+                rerum voluptatem illum itaque obcaecati facilis neque molestiae
+                quod suscipit voluptatibus. Debitis, totam? Odit quod, nihil
+                laudantium maiores recusandae sit perferendis?
+              </p>
+            </div>
+            <div
+              class="inline-flex items-center justify-center rounded-full text-green-400 bg-gray-800 flex-shrink-0"
+            ></div>
+          </div>
+          <div class="flex items-center lg:w-3/5 mx-auto sm:flex-row flex-col">
+            <div
+              class="inline-flex items-center justify-center rounded-full text-green-400 bg-gray-800 flex-shrink-0"
+            ></div>
+            <div class="flex-grow text-center mt-6 sm:mt-0">
+              <h2 class="text-green-500 text-lg title-font font-bold mb-2">
+                Open Source
+              </h2>
+              <p class="leading-relaxed text-base">
+                Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+                Excepturi commodi accusamus blanditiis officia molestias, ex in
+                incidunt quos qui explicabo, numquam voluptas doloremque rerum.
+                Dolorum aliquid dolores quidem nemo temporibus!
+              </p>
+            </div>
+          </div>
+          <button
+            class="flex mx-auto mt-20 bg-gradient-to-r from-green-400 to-green-600 border-0 py-2 px-8 focus:outline-none hover:from-cyan-500 hover:to-blue-500 rounded-full text-lg text-white"
+          >
+            <a href="#contributing">Contribute</a>
+          </button>
+        </div>
+      </section>
+    </div>
+    <div id="contributing">
```

Comment:
The items in this div being aligned to the left and the items in the following section being aligned to the center seems a bit weird.
This needs a bit of discussion.